components/service/glean/src/main/java/mozilla/components/service/glean/storages/EventsStorageEngine.kt

```diff
@@ -0,0 +1,83 @@
+package mozilla.components.service.glean.storages
```

Comment:
```suggestion
/* This Source Code Form is subject to the terms of the Mozilla Public
 * License, v. 2.0. If a copy of the MPL was not distributed with this
 * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
package mozilla.components.service.glean.storages
```

nit: We usually always add this license header to our files.